pkg/controller/accountclaim/accountclaim_controller.go

```diff
@@ -28,6 +32,7 @@ import (
 const (
 	AccountClaimed          = "AccountClaimed"
 	AccountUnclaimed        = "AccountUnclaimed"
+	OUName                  = "aws-account-operator-ou-mapping"
```

Comment:
Is the the name of the OU or of the configmap?

Reply:
configmap, I agree that the name should be more clear, changing.